install.sh

```diff
@@ -1,4 +1,44 @@
 #!/usr/bin/env bash
+# bash-it installer
+show_usage() {
+  echo -e "\n$0 : Install bash-it"
+  echo -e "Usage:\n$0 [arguments] \n"
+  echo "Arguments:"
+  echo "--help (-h): Display this help message"
+  echo "--silent (-s): Install default settings without prompting for input";
+  echo "--interactive (-i): Interactively choose plugins"
+  exit 0;
+}
+
+echo "Installing bash-it"
+
+for param in "$@"; do
+  shift
+  case "$param" in
+    "--help")        set -- "$@" "-h" ;;
+    "--silent")      set -- "$@" "-s" ;;
+    "--interactive") set -- "$@" "-i" ;;
+    *)               set -- "$@" "$param"
+  esac
+done
+
+OPTIND=1
+while getopts "hsi" opt
+do
+  case "$opt" in
+  "h") show_usage; exit 0 ;;
+  "s") silent=true ;;
+  "i") interactive=true ;;
+  "?") show_usage >&2; exit 1 ;;
+  esac
+done
+shift $(expr $OPTIND - 1)
+
+if [[ $silent ]] && [[ $interactive ]]; then
+  echo "Options --silent and --interactive are mutually exclusive. Please choose one or the other."
```

Comment:
I think it makes sense to print this in red color, like the other install error messages. The message should be surrounded by `\033[91m` and `\033[m` to make the installer messages consistent.